docs/DeveloperGuide.adoc

```diff
@@ -295,131 +304,434 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* *`|student|search a module based on keywords|
+|`* *`|diligent student|tag my modules so as to classify them better|
+|`* *`|student|fill my study plan with UEs too|
+|`* *`|student|rename UEs as their actual names|
+|`* *`|visual user|be able to visualise my modules in a GUI|
+|`* *`|experienced user| chain my commands together so that I can be more efficient|
+|`* *`|student who admires beauty| see different colours|
+|`* *`|forgetful student|attach more information to each semester|I will remember why I plan my modules this way
+|`* *`|fickle user|combine different semesters from different study plans into one new study plan (move semesters around)| I don’t have to repeat.
+|`* *`|student who wants to be a TA|indicate I will be TA-ing a module in a given semester|
+|`* *`|CS student with extra programmes| I want to verify that my study plan allows me to graduate with all the different requirements I have|
+|`*` |overachiever| I want to see how joining one of these Turing/von Neumann programmes affects my study plan|
+|`*` |student who wants to be a TA| indicate I will be TA-ing a module in a given semester|
+|`*` |CS student with extra programmes| I want to verify that my study plan allows me to graduate with all the different requirements I have|
+|`*` |CS student with friends| I want to plan modules with my friends | we can take the same modules every semester
+|`*` |student who cares about grades| I want to be able to analyse my CAP per module, semester, year and overall| I can be more aware of my grades
+|`*` |user who admires beauty| I want to set each module with a color 1-8 | I can customise the look of my study plan
+|`*` |user who admires beauty| I want to change the color theme | I can customise the look of my study plan
+|`*` |experienced user who admires beauty| I want to change specific colors with hex code | I can customise the look of my study plan
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
-_{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `ModBuddy` and the *Actor* is the `Student`, unless specified otherwise)
 
-[discrete]
-=== Use case: Delete person
+=== Use case: UC01 - Create study plan
+
+*MSS*
+
+1. User chooses to create a study plan.
+2. User enters the requested description.
+3. ModBuddy displays the new study plan.
+
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. User chooses not to enter a description.
+** 2a1. ModBuddy creates a new study plan with a default description.
```

Comment:
Is this format correct? Should you omit the bullets (the small circles at the beginning of a line) when you have already been using the number bullets?
You may add a [none] when there is an increase in the number of "*" in you code so as to omit the bullets